cli/args.go

```diff
@@ -53,9 +53,23 @@ func parseTerragruntOptionsFromArgs(args []string) (*options.TerragruntOptions,
 		WorkingDir: workingDir,
 		Logger: util.CreateLogger(""),
 		RunTerragrunt: runTerragrunt,
+		Env: parseEnvironmentVariables(os.Environ()),
 	}, nil
 }
 
+func parseEnvironmentVariables(environment []string) map[string]string {
+	environmentMap := make(map[string]string)
+
+	for i := 0; i < len(environment); i++ {
+		variableSplit := strings.SplitN(environment[i], "=", 2)
+		if (len(variableSplit) == 2) {
```

Comment:
In Go, you don't need parens around the if-statement clause. So this can be shortened to:

```go
if len(variableSplit) == 2 {
  ...
}
```